src/wallet/test/coinselector_tests.cpp

```diff
@@ -177,7 +177,7 @@ BOOST_AUTO_TEST_CASE(bnb_search_test)
     // Setup
     std::vector<CInputCoin> utxo_pool;
     CoinSet selection;
-    CoinSet actual_selection;
+    SelectionResult expected_result;
```

Comment:
Could 75672596d018f857013599b94f65efaf6446a825 perhaps be a scripted diff?

Reply:
Done